.env.example

```diff
@@ -1,2 +0,0 @@
-# RENAME TO .env AND POPULATE
```

Comment:
Maybe this could be reworded to `# COPY TO .env AND ASK TEAM MEMBER FOR VALUES` or something to make it clearer?

Reply:
Done